pkg/cmd/pipeline/delete.go

```diff
@@ -87,14 +113,19 @@ func checkOptions(opts *deleteOptions, s *cli.Stream, p cli.Params, pName string
 		return nil
 	}
 
-	fmt.Fprintf(s.Out, "Make sure you really want to delete pipeline %q (y/n): ", pName)
+	if opts.deleteAll {
+		fmt.Fprintf(s.Out, "Make sure you really want to delete pipeline and related resources (pipelineruns) %q (y/n): ", pName)
```

Comment:
Is it better getting all related resources and show these for user before deleting resources??